app/src/main/java/com/sdp/movemeet/RegisterActivity.java

```diff
@@ -70,23 +64,23 @@ protected void onCreate(Bundle savedInstanceState) {
 
     public void registerUser(View view) { // when the user clicks on the "REGISTER" button, we validate his data
 
-        email = mEmail.getText().toString().trim();
-        password = mPassword.getText().toString().trim();
-        fullName = mFullName.getText().toString();
-        phone = mPhone.getText().toString();
+        emailString = emailEditText.getText().toString().trim();
```

Comment:
Nice! I think this refactoring was mentioned in the code review.

Reply:
Yes, this goes in the direction of what we have discussed on Tuesday about our convention for the name of variables.